compiler/rustc_session/messages.ftl

```diff
@@ -14,6 +14,8 @@ session_crate_name_empty = crate name must not be empty
 
 session_crate_name_invalid = crate names cannot start with a `-`, but `{$s}` has a leading hyphen
 
+session_embed_source_insufficient_dwarf_version = `-Zembed-source=y` requires at least `-Z dwarf-version=5` while session uses version {$dwarf_version}
```

Comment:
I'm not sure "session" is going to be meaningful to users, maybe just "but dwarf version is"?